src/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -20,6 +20,7 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="AllocationHelper.cs" />
+    <Compile Include="ReadOnlyBuffer\ReadOnlySequenceTests.cs" />
```

Comment:
nit: Can you please move all the ReadOnlyBuffer\* test files to the ItemGroup which has the other ones here https://github.com/dotnet/corefx/pull/29262/files#diff-00a03683c006a09943017a6311ec51c9R90? Keep it alphabetical order.

Move these down with the rest:
```csproj
<Compile Include="ReadOnlyBuffer\ReadOnlySequenceTests.cs" />
<Compile Include="ReadOnlyBuffer\ReadOnlySequenceTests.Common.char.cs" />
<Compile Include="ReadOnlyBuffer\ReadOnlySequenceTests.Common.byte.cs" />
<Compile Include="ReadOnlyBuffer\ReadOnlySequenceTests.Default.cs" />
<Compile Include="ReadOnlyBuffer\ReadOnlySequenceTests.Empty.cs" />
```

Thanks.